src/test/java/nl/tudelft/ewi/devhub/server/database/AssignTAsTest.java

```diff
@@ -0,0 +1,117 @@
+package nl.tudelft.ewi.devhub.server.database;
+
+import nl.tudelft.ewi.devhub.server.database.controllers.AssignedTAs;
+import nl.tudelft.ewi.devhub.server.database.entities.AssignedTA;
+import nl.tudelft.ewi.devhub.server.database.entities.Assignment;
+import nl.tudelft.ewi.devhub.server.database.entities.Group;
+import nl.tudelft.ewi.devhub.server.database.entities.User;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Created by sayra on 07/06/2017.
+ */
+public class AssignTAsTest {
+
+    Group group1;
+    Group group2;
+    Group group3;
+    Group group4;
+    User ta1;
+    User ta2;
+    List<Group> groups;
+    List<Group> singleGroup;
+    List<User> teachingAssistants;
+    Assignment assignment;
+    AssignedTA assignedTA1Group2;
+    AssignedTA assignedTA1Group3;
+    AssignedTA assignedTA1Group1;
+    AssignedTA assignedTA1Group4;
+    AssignedTA assignedTA2Group1;
+    AssignedTA assignedTA2Group4;
+    List<AssignedTA> assignedTAS;
```

Comment:
I moved the construction of AssignedTAs to their respective tests to improve readability